.platform.app.yaml

```
@@ -0,0 +1,99 @@
+name: sylius
+
+type: php:8.1
```

I think it should be either minimal (8.0 for 1.12) version or latest version.

```suggestion
type: php:8.3
```